lib/entry.js

```diff
@@ -14,14 +14,16 @@ function Entry(exports) {
   this.exports = exports;
   // The number of times this.runSetters has been called.
   this.runCount = 0;
-  // Setters for assigning to local variables in parent modules.
-  this.setters = new FastObject;
   // Getters for local variables exported from this module.
-  this.getters = new FastObject;
+  this.getters = Object.create(null);
+  // Setters for assigning to local variables in parent modules.
+  this.setters = Object.create(null);
 }
 
 var Ep = Entry.prototype;
 
+Ep.__proto__ = null;
```

Comment:
`__proto__` is supported in IE 11+ and all other browsers. It won't hurt older ones though.
This avoids using ES6 `Object.setPrototypeOf`.